web/packages/teleport/src/Locks/NewLock.tsx

```diff
@@ -110,22 +113,22 @@ export function NewLockContent({
       <Flex justifyContent="space-between">
         <Box width="164px" mb={4} data-testid="resource-selector">
           <Select
-            value={selectedTargetType}
+            value={selectedDropdownOption}
             options={lockTargets}
-            onChange={(o: LockTarget) => setSelectedTargetType(o)}
+            onChange={(o: DropdownOption) => setSelectedDropdownOption(o)}
```

Comment:
Similarly, the Teleport resource this controls is called a `LockTarget`. I don't see how renaming that to something less specific is an improvement.